src/app/pages/subscription/subscription-routing.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { SubscriptionComponent } from './subscription/subscription.component';
+
+const routes: Routes = [{ path: '', component: SubscriptionComponent }];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule],
+})
+export class SubscriptionRoutingModule {}
```

Comment:
explain why a new module is needed instead of a single component (not everything needs to be bundled into modules).
(IMO: remove module and add the page as a simple component)

Reply:
we need to load *one* component per module. Where does that increases the payload that much? 
Data is served from core-service on initialization and not directly with the component.